src/reducers/reducer.js

```diff
@@ -16,6 +16,7 @@ function getSavedTaskKeys(key) {
 function writeChanges(tasks, typeKey) {
   const existingTaskKeys = getSavedTaskKeys(typeKey)
   const newTaskKeys = tasks.map(task => task.storageKey)
+  console.info(typeKey, newTaskKeys)
```

Comment:
I intentionally added some `console.info` calls thinking they could stick around. If you see a stray `console.log`, that was meant as temporary debugging and should be removed!

This is a pattern I've followed in other projects of mine, notably Chrome extensions. If you open the browser inspector, it's nice to see a log of the actions the app has been taking. Helps to reconstruct what may have gone wrong if there's a bug.